src/hotspot/share/compiler/compilationFailureInfo.cpp

```diff
@@ -33,12 +33,12 @@
 #include "compiler/compilationFailureInfo.hpp"
 #include "compiler/compileTask.hpp"
 #ifdef COMPILER2
```

Comment:
Conditional includes are supposed to follow unconditional in a section.
Out of scope for this PR?

Reply:
Yep. From the PR description:
```
The tool does nothing about re-ordering blocks of conditional includes vs unconditional includes. I briefly looked into that but it gets very complicated, very quickly. That kind of re-ordering will have to continue to be done manually for now.
```